asserts/gpgkeypairmgr.go

```diff
@@ -183,3 +223,83 @@ func (gkm *GPGKeypairManager) sign(fingerprint string, content []byte) ([]byte,
 	}
 	return out, nil
 }
+
+type gpgKeypairInfo struct {
+	pubKey      PublicKey
+	fingerprint string
+}
+
+func (gkm *GPGKeypairManager) findByName(name string) (*gpgKeypairInfo, error) {
+	stop := errors.New("stop marker")
+	var hit gpgKeypairInfo
```

Comment:
@cjwatson sorry there's a typo in my suggestion, my point is having the var be a pointer (*), not changing the name, but I fatfingered a capital P in there